lms/djangoapps/certificates/views/webview.py

```diff
@@ -100,13 +100,10 @@ def _update_certificate_context(context, user_certificate, platform_name):
 
     # Translators:  The format of the date includes the full name of the month
     course = get_course_by_id(user_certificate.course_id) if user_certificate.course_id else None
-
-    if all([
-        course,
-        course.certificate_available_date,
-        course.self_paced is False,
-        course.certificate_available_date < datetime.datetime.now(),
-    ]):
+    if course and
```

Comment:
You'll need to either wrap this in parens or use backslashes to break it up across lines.
e.g.
```python
if (course and
    course.certificate_available_date and
    course.self_paced is False and
    course.certificate_available_date < datetime.datetime.now()
):
```